tests/send-to-helix-workload-tests.targets

```diff
@@ -51,18 +64,21 @@
            Text="Could not find dotnet-coverage tool. %24(_DotNetCoverageToolPath)=$(_DotNetCoverageToolPath)" />
 
     <ItemGroup>
+      <_E2ETestScenarios Include="basicservices" />
+      <_E2ETestScenarios Include="cosmos" />
+
       <HelixCorrelationPayload Include="$(ArtifactsBinDir)dotnet-latest" Destination="dotnet-latest" />
       <HelixCorrelationPayload Include="$(ArtifactsShippingPackagesDir)" Destination="built-nugets" />
 
-      <HelixWorkItem Include="@(_DefaultWorkItemsWorkloadTests -> '%(FileName)')">
-        <PayloadArchive>%(Identity)</PayloadArchive>
-        <PreCommands Condition="'$(OS)' == 'Windows_NT'">set &quot;TEST_NAME=%(FileName)&quot;</PreCommands>
-        <PreCommands Condition="'$(OS)' != 'Windows_NT'">export &quot;TEST_NAME=%(FileName)&quot;</PreCommands>
+      <HelixWorkItem Include="@(_E2ETestScenarios -> '$(_WorkItemName)-%(Identity)')">
+        <PayloadArchive>$(_E2ETestsArchivePath)</PayloadArchive>
+        <PreCommands Condition="'$(OS)' == 'Windows_NT'">set &quot;TEST_NAME=$(_WorkItemName)&quot;</PreCommands>
+        <PreCommands Condition="'$(OS)' != 'Windows_NT'">export &quot;TEST_NAME=$(_WorkItemName)&quot; &amp;&amp; export TEST_SCENARIO=%(Identity)</PreCommands>
         <Command>$(_TestRunCommand)</Command>
         <Timeout>$(_WorkItemTimeoutForWorkloadTests)</Timeout>
 
         <!-- Download results file so coverage files can be extracted -->
-        <DownloadFilesFromResults>logs/%(FileName).cobertura.xml</DownloadFilesFromResults>
+        <DownloadFilesFromResults>logs/Aspire.EndToEnd.Tests.cobertura.xml</DownloadFilesFromResults>
```

Comment:
Isn't this name going to conflict between multiple _E2ETestScenarios ?

Reply:
hmph yeah:
```
Copying /mnt/vss/_work/1/s/artifacts/helixresults/cc085e87-12fa-4085-8946-dc76e4f0f4c1/Aspire.EndToEnd.Tests-cosmos/logs/Aspire.EndToEnd.Tests.cobertura.xml to /mnt/vss/_work/1/s/artifacts/CodeCoverage/Aspire.EndToEnd.Tests.cobertura.xml
File /mnt/vss/_work/1/s/artifacts/helixresults/cc085e87-12fa-4085-8946-dc76e4f0f4c1/Aspire.EndToEnd.Tests-scenario0/logs/Aspire.EndToEnd.Tests.cobertura.xml already exist at /mnt/vss/_work/1/s/artifacts/CodeCoverage/Aspire.EndToEnd.Tests.cobertura.xml
File /mnt/vss/_work/1/s/artifacts/helixresults/cc085e87-12fa-4085-8946-dc76e4f0f4c1/Aspire.EndToEnd.Tests-sqlserver/logs/Aspire.EndToEnd.Tests.cobertura.xml already exist at /mnt/vss/_work/1/s/artifacts/CodeCoverage/Aspire.EndToEnd.Tests.cobertura.xml
```

Reply:
fixed.